mappings/net/minecraft/client/render/entity/model/AxolotlEntityModel.mapping

```diff
@@ -1,4 +1,44 @@
 CLASS net/minecraft/class_5772 net/minecraft/client/render/entity/model/AxolotlEntityModel
+	COMMENT Represents the model of an {@linkplain AxolotlEntity}.
+	COMMENT
+	COMMENT <div class="fabric">
+	COMMENT <table border=1>
+	COMMENT <caption>Model parts of this model</caption>
+	COMMENT <tr>
+	COMMENT   <th>Part Name</th><th>Parent</th><th>Corresponding Field</th>
```

Comment:
Hmm, is it possible to squeeze this into only two columns? Like for a top gills, we have it as:
```
<td>{@link #topGills top_gills}</td><td>{@code head}</td>
```
Or use `@code` for parts that aren't in fields. The rendered effect is a code block with a link to that field.

Reply:
My issue with that is it prevents the linking of the key constants.

Reply:
Yeah, only discovered that later.